unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -691,6 +691,7 @@ public class TimeReportPayload
     [DllImport("__Internal")] public static extern void MessageFromEngine(string type, string message);
     [DllImport("__Internal")] public static extern string GetGraphicCard();
     [DllImport("__Internal")] public static extern bool CheckURLParam(string targetParam);
+    [DllImport("__Internal")] public static extern string GetURLParam(string targetParam);
```

Comment:
Also, why is this needed.

Reply:
Cool, maybe we should state this information somehwere in the code itself. To make it clear than in normal production environment we dont care about URLs, we just do what kernel says.

Reply:
May I suggest to send this information via KernelConfig instead?

Reply:
I removed them in [this commit](https://github.com/decentraland/unity-renderer/pull/2345/commits/da6dc59b49b8fdbeb15cf789a097d3df67f4eef2) and [created this issue](https://github.com/decentraland/unity-renderer/issues/2519) for the future